### ISSUE TITLE
It has in its roadmap that the IpsCards have theme settings (light or dark)

### ISSUE THREAD

**Issue by rcoproc** [andreciornavei/flutter-input-sheet#1]
Congratulations, its components worked very well here on my Android Action One. 

![image](https://user-images.githubusercontent.com/7529472/82006995-78ae0d00-963f-11ea-9904-c97a3963c9e9.png)

I'll see if I can test it on IOS and give you feedback.

I liked the practical way of using them in forms. 

Hug and good luck there.

**Comment by andreciornavei** [comment]
Great !! Happy for helping.

Now, the purpose of this package is implement the Input in BottomSheets, the Basic components was just to implement an example component to make your forms beauty. 

The `IpsLabel`, `IpsIcon`, `IpsValue` and `IpsError`, is a simple Text and Icon Widget and i believe if you want to custom it, you can create your own widgets. 

But, of corse, like the `IpsCard` is the owner of all the Widgets above, make sense to make it theme editable.  So like the `IpsCard` is a FlatButton, i can to add `color`,  `highlightColor`, `splashColor`and `shape` params to override the default implementation, then you can be free to implement dark mode like code below.

What do you think?

```dart
IpsCard(
      label: IpsLabel("Name"),
      value: IpsValue(_name ?? "Touch to edit..."),
      icon: IpsIcon(FeatherIcons.user),
      error: IpsError(_errors['_name']),
      color: Color(0xff333333),
      highlightColor: Color.fromARGB(10, 255, 255, 255),
      splashColor: Color.fromARGB(10, 255, 255, 255),
      shape: RoundedRectangleBorder(
            side: BorderSide(color: Colors.black, width: 1),
            borderRadius: BorderRadius.all(
                  Radius.circular(8),
             ),
       ),
      onClick: (){}
);
```

**Comment by andreciornavei** [comment]
Implemented on release 0.1.0
